src/ast/nodes/ClassExpression.ts

```diff
@@ -1,11 +1,16 @@
 import ClassNode from './shared/ClassNode';
 import Scope from '../scopes/Scope';
+import { ObjectPath } from '../variables/VariableReassignmentTracker';
+import ExecutionPathOptions from '../ExecutionPathOptions';
+import { Expression } from './shared/Expression';
 
-export default class ClassExpression extends ClassNode {
+export default class ClassExpression extends ClassNode implements Expression {
```

Comment:
This then throws me out as I'd have expected `ExpressionNode` to be the pattern here...

Reply:
This is more of a relic as `ClassNode` already implements `ExpressionNode`. I'll remove it.